float64 dist1
float64 dist2
float64 dist3
float64 dist4
float64 dist5
float64 dist6
float64 dist7
float64 dist8
float64 left_wheel
float64 right_wheel
float64 imu_r
float64 imu_p
float64 imu_y

